### PR TITLE
edit apriori, dxe and sm6225.fog files

### PR DESCRIPTION
**Pull request by Statzar**
### Description
Changed path to patched HALIOMMU and Sdcc
[Description of the changes you are submitting]

### Checklist

* [x] Have you tested the change you are submitting?
* [x] Is the commits history nice and clean?

### CODE REVIEW

File: Platform/Qualcomm/sm6225/sm6225.fog.dsc [hunk 1]
```diff
@@ -62,8 +62,8 @@
 
   # Ported from SurfaceDuoPkg
   AslUpdateLib|Silicon/Qualcomm/QcomPkg/Library/DxeAslUpdateLib/DxeAslUpdateLib.inf
-  SerialPortLib|Silicon/Qualcomm/QcomPkg/Library/FrameBufferSerialPortLib/FrameBufferSerialPortLib.inf
-  #SerialPortLib|MdePkg/Library/BaseSerialPortLibNull/BaseSerialPortLibNull.inf
+  #SerialPortLib|Silicon/Qualcomm/QcomPkg/Library/FrameBufferSerialPortLib/FrameBufferSerialPortLib.inf
+  SerialPortLib|MdePkg/Library/BaseSerialPortLibNull/BaseSerialPortLibNull.inf
```
Comment (MollySophia) [inline Platform/Qualcomm/sm6225/sm6225.fog.dsc:66]: Just curious. Why is here changing to BaseSerialPortLibNull? :P

**Review by MollySophia — APPROVED** [review]

File: Platform/Qualcomm/sm6225/sm6225.fog.dsc (hunk 1 above)
Comment (BigfootACA) [inline Platform/Qualcomm/sm6225/sm6225.fog.dsc:66]: SerialPortLib should not add here

**Review by BigfootACA — CHANGES_REQUESTED** [review]

File: Platform/Qualcomm/sm6225/sm6225.fog.dsc (hunk 1 above)
Comment (Statzar) [reply Platform/Qualcomm/sm6225/sm6225.fog.dsc]: Okay

Comment (Statzar) [reply Platform/Qualcomm/sm6225/sm6225.fog.dsc]: To exclude log output

**Pull request closed without merging**